backend/connector_v2/models.py

```diff
@@ -111,17 +111,42 @@ def __str__(self) -> str:
             f" workflow: {self.workflow})"
         )
 
+    def get_connector_metadata_bytes(self):
```

Comment:
@muhammad-ali-e why isn't this required for adapters?

Reply:
In this case, I merged the binary and normal fields to avoid having two fields serving the same purpose. I tested some scenarios, but more testing is needed. For the adapter, I didn't merge them due to time constraints and the complexity of handling multiple cases. We can merge them there later if necessary. This function is specifically used to convert memoryview to bytes due to the introduction of the new field.